test/e2e/live_iso_test.go

```diff
@@ -30,7 +32,17 @@ var _ = Describe("Live-ISO", Label("required", "live-iso"), func() {
 	)
 
 	BeforeEach(func() {
-		imageURL = e2eConfig.GetVariable("IMAGE_URL")
+		// Check what kind of BMC we are dealing with
+		// It may be *possible* to boot a live-ISO over (i)PXE, but there are severe limitations.
+		// Therefore we skip the test if it doesn't support ISO preprovisioning images.
+		// See https://docs.openstack.org/ironic/latest/admin/ramdisk-boot.html
+		accessDetails, err := metal3bmc.NewAccessDetails(bmc.Address, false)
+		Expect(err).NotTo(HaveOccurred())
+		if !accessDetails.SupportsISOPreprovisioningImage() {
```

Comment:
I could use a second opinion here. Does it make sense to skip the test for ipmi and redfish? Is this the right thing to check?
Based on the docs I linked in the comment above, I think it isn't really worth it to test live-ISO in non-virtualmedia cases and the test currently fails for them. This is why I added the skip.

Reply:
Technically, you can boot an ISO via iPXE. We added this to BMO purely accidentally, but when a validation was added that blocked it, some customers complained. Apparently, people use this in production.

Reply:
But would you say we need to test it? It seems we would need some special ISO to get something useful with iPXE.

> This feature, when utilized with the ipxe boot_interface, will only allow a kernel and ramdisk to be booted from the supplied ISO file. Any additional contents, such as additional ramdisk contents or installer package files will be unavailable after the boot of the Operating System. Operators wishing to leverage this functionality for actions such as OS installation should explore use of the standard ramdisk deploy_interface along with the instance_info/kernel_append_params setting to pass arbitrary settings such as a mirror URL for the initial ramdisk to load data from. This is a limitation of iPXE and the overall boot process of the operating system where memory allocated by iPXE is released.

Reply:
IMHO it's probably not worth testing it at this stage
we could add a TODO for the future as a reminder in case this become more impactful, but I doubt it